static/styles/lib/colors.scss

```diff
@@ -1,2 +1,3 @@
 $colorBeige: #e6e2e2;
 $colorGrey: rgba(232, 232, 232, 0.5);
+$colorDarkerGrey: #555555;
```

Comment:
Die Farben fürs StandardTheme sind jetzt alle in theme/default/style.scss - sollte die auch dahin?

Reply:
ist ja ne Farbe und keine sematische Zuordnung .. also eher nein würde ich sagen